internal/app/singularity/remote_use.go

```diff
@@ -12,12 +12,35 @@ import (
 	"github.com/sylabs/singularity/internal/pkg/remote"
 )
 
+func syncSysConfig(cUsr *remote.Config, sysConfigFile string) error {
+	// opening system config file
+	f, err := os.OpenFile(sysConfigFile, os.O_RDONLY, 0600)
+	if err != nil {
+		return fmt.Errorf("while opening remote config file: %s", err)
```

Comment:
I think you should check `IsNotExist()` and just return without syncing. Otherwise you must have a system config set in order to use user remotes.